examples/with-federated-passkeys/src/pages/_document.tsx

```diff
@@ -0,0 +1,19 @@
+import Document, { Html, Head, Main, NextScript } from "next/document";
+
+class MyDocument extends Document {
```

Comment:
I think I commented on this previously, but it's a minor thing: let's rename this "FederatedPasskeysDemo", "TurnkeyDemo" or something similar?